scio-core/src/main/scala/com/spotify/scio/values/ApproxFilter.scala

```diff
@@ -0,0 +1,53 @@
+package com.spotify.scio.values
+import com.spotify.scio.annotations.experimental
+import com.spotify.scio.coders.Coder
+
+/**
+ * A read only container for collections which can be used to check approximate
+ * membership queries.
+ *
+ * Constructors for [[ApproxFilter]] are defined using [[ApproxFilterBuilder]]
+ */
+trait ApproxFilter[-T] extends (T => Boolean) {
```

Comment:
This is a weird use for `A => B`, is it really necessary vs simply `extends Serializable` and `def apply(t: T): Boolean`?

Reply:
I am not sure why I did this. I'll remove this. I think just having apply may be a bit confusing, let's have the `mayBeContains` and be explicit.